giskard-server/src/main/java/ai/giskard/service/dto/ml/DatasetDTO.java

```diff
@@ -5,25 +5,13 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
-import javax.validation.constraints.NotNull;
-
 @Getter
 @Setter
 @NoArgsConstructor
-public class DatasetDTO {
```

Comment:
hmm, I don't think it makes sense that dataset inherits model. I'd be better to actually recreate the original class structure with a `ProjectFileDTO` inheritence

Reply:
yes wanted to refactor this by recreating the abstract class